renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -45,6 +45,34 @@ static std::string ToHumanStr(const VkAttachmentStoreOp &el)
   END_ENUM_STRINGISE();
 }
 
+void WrappedVulkan::AddImplicitResolveResourceUsage(uint32_t subpass)
+{
+  ResourceId rp = m_BakedCmdBufferInfo[m_LastCmdBufferID].state.renderPass;
+  ResourceId fb = m_BakedCmdBufferInfo[m_LastCmdBufferID].state.framebuffer;
+
+  VulkanCreationInfo::Framebuffer fbinfo = m_CreationInfo.m_Framebuffer[fb];
+  VulkanCreationInfo::RenderPass rpinfo = m_CreationInfo.m_RenderPass[rp];
```

Comment:
These `fbinfo` / `rpinfo` shouldn't be copied as they're large structures, since we only need read-only access then const references is preferred. It also seems like fbinfo isn't used here?